ext/bin/pdbbox-init

```diff
@@ -36,20 +37,27 @@ while test $# -gt 0; do
             pgport="$1"
             shift
             ;;
+        --bind-addr)
+            shift
+            test ! -z $# || misuse
+            bind_addr[${#bind_addr[@]}]="$1"
+            shift
+            ;;
         *)
             misuse
     esac
 done
 test "$pdbbox" || misuse
 test "$pgbin" || misuse
 test "$pgport" || misuse
+test "$bind_addr" || misuse
 
 export PGBOX="$pdbbox/pg"
 export PGUSER=postgres
 
 mkdir "$pdbbox"
 
-pgbox init --pgbin "$pgbin" --port "$pgport" -- -E UTF8 --locale=C
+pgbox init --pgbin "$pgbin" --port "$pgport" --bind-addr "${bind_addr[@]/#/}" -- -E UTF8 --locale=C
```

Comment:
Hmm, interesting, what does the `/#/` do?  And I suspect this might actually need to build all the args in an array or something, i.e. with alternating `--bind-addr` and ADDR values, so it'll handle multiple addresses.  

Either way, might also be worth changing at least one of the pgbox-init calls in our tests to specify 2 --bind-addr values, so we'll implicitly test that case.  I suppose it might be fine to just specify `localhost` twice, or if pg won't care if the second address is actually available everywhere we test (it may not -- iirc it may just warn), suppose we might just add ip6-localhost in one of our invocations...

Reply:
This builds it into: `--bind-addr localhost ip6-localhost`.

Reply:
At this point, I'm going to defer to you on this PR and feature entirely. Make any changes you feel necessary to implement it your way. I've reworked this enough.